libs/CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ if(WITH_FMT)
     add_subdirectory(fmt)
 endif()
 
+option(WITH_CLIP  "Build clip.hdll" ON)
```

Comment:
You call it `clipboard.hdll` later.

Reply:
Yeah, that's because of the way I'm including the library (with CMake's add_subdirectory atm, don't know if that's the best way, but there isn't binary distributions for the library, so I thought I'd just attach it to the CMake build process itself). It creates a conflict in the build, because you have a subproject with the same name as the hl lib -project. I don't know if you can bypass that somehow, though.

Reply:
Ah, yeah, had a brainfart, finally looked at what you were actually pointing at :) Yeah that was from the time I called it "clip.hdll" and not "clipboard.hdll".